utils/dockerutil.py

```diff
@@ -603,6 +607,15 @@ def image_name_resolver(self, image):
         else:
             return image
 
+    def label_extractor(self, ctr, lbl_to_tags):
+        """Returns a list of tags based on a container and a label name list"""
+        tags = []
+        labels = ctr.get('Config', {}).get('Labels', {})
+        for lbl_name, lbl_val in labels.iteritems():
```

Comment:
Just to be extra-safe, let's add a `if not labels: return tags`, I had the issue of a Config/Env being present but None instead of missing. See https://github.com/DataDog/dd-agent/pull/3528

Reply:
good catch!